internal/bus/chainsubscriber.go

```diff
@@ -210,10 +219,10 @@ func (s *chainSubscriber) applyChainUpdate(tx sql.ChainUpdateTx, cau chain.Apply
 
 	// v2 contracts
 	var revisedContracts []types.V2FileContractElement
-	cau.ForEachV2FileContractElement(func(fce types.V2FileContractElement, created bool, rev *types.V2FileContractElement, res types.V2FileContractResolutionType) {
-		if created {
-			revisedContracts = append(revisedContracts, fce) // created
-		} else if rev != nil {
+	cau.ForEachV2FileContractElement(func(fce types.V2FileContractElement, _ bool, rev *types.V2FileContractElement, res types.V2FileContractResolutionType) {
```

Comment:
does this mean `created` is redundant in the `core` package?

Reply:
Kinda. It is only `true` if both `rev` and `res` are `nil`. Otherwise there will always be one of either set.